src/commands/app/deploy.js

```diff
@@ -171,9 +152,13 @@ Deploy.flags = {
   'skip-actions': flags.boolean({
     description: 'Skip action build & deploy'
   }),
+  'force-build': flags.boolean({
+    description: 'Forces a build even if one already exists',
+    exclusive: ['skip-build'],
+    default: false
```

Comment:
should there be an "allowNo: true" property here?

Reply:
The default is false here. So i think we don't need it. It is there in `aio app build` because default for `force-build` is true there. Also we are calling `build` from `deploy` with a `no-force-build` flag when `force-build` is not specified.